certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json

```diff
@@ -437,19 +450,23 @@
                     "properties": {
                       "file_name": {
                         "type": "string",
-                        "description": "file_name"
+                        "description": "file_name",
+                        "title": "file name"
                       },
                       "diseases_found": {
                         "type": "string",
-                        "description": "diseases_found"
+                        "description": "diseases_found",
+                        "title": "diseases found"
                       },
                       "cpt_code": {
                         "type": "string",
-                        "description": "cpt_code"
+                        "description": "list of cpt codes",
+                        "title": "cpt code"
                       },
                       "cpt_desc": {
                         "type": "string",
-                        "description": "cpt_desc"
+                        "description": "cpt_code_description",
+                        "title": "cpt desc"
```

Comment:
Hi@sriyen-msft,
Sorry for the mistake, I have corrected it please do check once,
},
"cpt_desc": {
"type": "string",
"description": "cpt_code_description",
"title": "cpt desc"
},